docs/integrations/sources/lemlist.md

```diff
@@ -31,10 +31,11 @@ The Lemlist connector should not run into Lemlist API limitations under normal u
 
 ### Requirements
 
-* Lemlist API key
+- Lemlist API key
 
 ## Changelog
 
-| Version | Date | Pull Request | Subject |
-| :--- | :--- | :--- | :--- |
-| 0.1.0 | 2021-10-14 | [7062](https://github.com/airbytehq/airbyte/pull/7062) | Initial Release |
+| Version | Date       | Pull Request                                           | Subject         |
+| :------ | :--------- | :----------------------------------------------------- | :-------------- |
+| 0.1.1   | ?          | [unknown](https://github.com/airbytehq/airbyte/pulls)  | Bump Version    |
```

Comment:
Not sure I'd link to empty pull request page here, rather just "Unknown" (looks clearer than ?) for both date and pull request, no link